documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,255 @@ You may obtain a copy of the License at
 
 Pravega is an open source storage primitive implementing **Streams** for continuous and unbounded data.
```

Comment:
Rephrase this sentence: `Pravega is an open source storage system implementing **Streams** as first-class primitive for continuous and unbounded data.`